spec/fogbugz.spec.js

```diff
@@ -60,8 +105,8 @@ describe('fogbugz module', function () {
     });
   });
 
-  describe('logoff method', function (done) {
-    it('should fail w/o presence of token', function () {
+  describe('logoff method', function () {
+    it('should fail w/o presence of token', function (done) {
```

Comment:
thanks for this.  pretty old module.  if I had my druthers I'd rewrite it in mocha, but whatever